readthedocs/projects/models.py

```diff
@@ -211,8 +211,11 @@ class Project(models.Model):
     privacy_level = models.CharField(
         _('Privacy Level'), max_length=20, choices=constants.PRIVACY_CHOICES,
         default=getattr(settings, 'DEFAULT_PRIVACY_LEVEL', 'public'),
-        help_text=_('(Beta) Level of privacy that you want on the repository. '
-                    'Protected means public but not in listings.'))
+        help_text=_('(Beta) Level of privacy that you want on the project. '
```

Comment:
Here I changed the word _repository_ to _project_.

Reply:
Maybe the "Beta" work can be removed since it's not in beta anymore and I understood that you realized that it worked properly but what's is wrong is the docs, right?

Reply:
I also think this isn't beta anymore.